simphony/cuds/mesh.py

```diff
@@ -676,6 +684,33 @@ def has_cells(self):
         """
         return len(self._cells) > 0
 
+    def count_of(self, item_type):
+        """ Return the count of item_type in the container.
+
+        Parameter
+        ---------
+        item_type : CUBA
+            The CUBA enum of the type of the items to return the count of.
+
+        Returns
+        -------
+        count : int
+            The number of items of item_type in the container.
+
+        Raises
+        ------
+        ValueError :
+            If the type of the item is not supported in the current
+            container.
+
+        """
+
+        try:
+            return len(self._items_count[item_type]())
+        except:
```

Comment:
I think there is a confusion. My comment is about the bare `try.. except:` which should be always avoided unless we really want to ignore all and any type of errors which is not the case.

What I am proposing is:

```python
try:
      return len(self._items_count[item_type]())
except KeyError:
      raise ValueError(....)
```